src/main/java/com/julienvey/trello/impl/domaininternal/StringValueHolder.java

```diff
@@ -0,0 +1,28 @@
+package com.julienvey.trello.impl.domaininternal;
+
+import java.util.Objects;
+
+public class StringValueHolder {
```

Comment:
let's see if there is a way to implement this feature without adding a wrapper around a string

Reply:
We could use `java.util.Map`.